fawltydeps/packages.py

```diff
@@ -194,32 +210,54 @@ def lookup_package(self, package_name: str) -> Optional[Package]:
         return self.packages.get(Package.normalize_name(package_name))
 
 
+class IdentityMapping(BasePackageResolver):
+    """An imperfect package resolver that assmues package name == import name.
```

Comment:
It looks like the lyrical subject has strong feelings about this mapping ;)
It is a joy to read those docstrings :D

```suggestion
    """An imperfect package resolver that assumes package name == import name.
```

Reply:
Haha. True, but I believe I can defend that assertion: the identity mapping is _objectively_ worse (more imperfect) than the other mappings we want to use, and that is exactly why we put it at the bottom of our stack. There is no plausible scenario in which placing the identity mapping at the top of the stack would yield better mapping results.